hexrd/ui/hexrd_config.py

```diff
@@ -716,6 +716,10 @@ def default_detector(self):
         return copy.deepcopy(
             self.default_config['instrument']['detectors']['ge1'])
 
+    def detector_pixel_size(self, detector_name):
+        detector = self.detector(detector_name)
+        return detector.get('pixels', {}).get('size', {}).get('value', [1, 1])
```

Comment:
@bnmajor is [1, 1] here the default pixel [row, col] size?  If so, we can make it [0.1, 0.1] to be more appropriate for the LLNL data.

Reply:
Sounds good, it has been updated.